src/views/Activity.vue

```diff
@@ -6,14 +6,12 @@ div
 
   p(style="color: red;") {{ errormsg }}
 
-  h4(style="color: red;")
-    | This is currently work in progress and is known to have issues (such as
-    a(href='https://github.com/ActivityWatch/aw-webui/issues/22')  needing a date selector
-    | ,
-    a(href='https://github.com/ActivityWatch/activitywatch/issues/72')  timezone issues
-    | , etc.) - see
-    a(href='https://github.com/ActivityWatch/aw-webui/issues')  all issues
-    | .
+  //div.alert.alert-info(role="alert")
```

Comment:
I wanted to keep the code in case we want to use it in other places.

Why not remove a link to the timezone issue? I fixed the basic issue and we settled that the remaining issues involving timezones won't be solved for quite a while, same for the datepicker.

The 'all issues' link isn't of much use since there aren't many issues that users need to concern themselves with.